sklearn/discriminant_analysis.py

```diff
@@ -245,6 +245,7 @@ class LinearDiscriminantAnalysis(BaseEstimator, LinearClassifierMixin,
     >>> print(clf.predict([[-0.8, -1]]))
     [1]
     """
+
```

Comment:
Looks like the docstring doesn't explain that ``store_covariance`` only makes a difference if ``solver='svd'``